runtime/doc/editorconfig.txt

```diff
@@ -86,4 +86,11 @@ Example: >lua
       vim.b[bufnr].foo = val
     end
 <
+
+EditorConfig overrides filetype plugins           *editorconfig-vs-ftplugin*
```

Comment:
I don't think this needs a separate section or its own tag. I would just update the sentence near the beginning of the help file.

For example, change "When a file is opened" to "When a file is opened, after running |ftplugin|s,".

Reply:
> For example, change \"When a file is opened\" to \"When a file is opened, after running |ftplugin|s,\".

Ooh, nice. I have much to learn about technical documentation.